content/response-to-reviewers.md

```diff
@@ -41,6 +41,11 @@ This a totally minor point, but it might help to attract new people.
 > You need to have a long-term archive (e.g. Zenodo) of manubot (just in case GitHub closes tomorrow).
 Or you can use a Software Heritage identifier to make sure manubot can still be found in a few years.
 
+This feedback was discussed in [GH135](https://github.com/greenelab/meta-review/issues/135) and addressed in [GH159](https://github.com/greenelab/meta-review/pull/159).
```

Comment:
Do we want to link to issues and pull requests for all responses?  We could, but I haven't done it for the other minor issues I closed.

Reply:
Yes, I was imagining we could use a sentence like this on every issue. I think it is helpful for viewers to see exactly where a comment was addressed. In addition, the issue and PRs often have additional comments that provide more information on why we made the changes we did.